### PR TITLE
Created add_payment_receipt service

### PR DESCRIPTION
**Pull request by johnwu93**
Added a new graphql service that let's you add payments manually into the transaction database.

Here is a getter/setter test using GraphQL:
# Mutation
## Command
```
mutation {AddPaymentReceipt (
  input: {
  payment:"AQEBAQEBAgEgaSF6MHmQgJThESHQQjVKfB9VtkgsoaUeGyUN/R7Q7vkBAAEBKDzug2KYQifipiVGHpdTtWAXN5fh+qOdXLBZFweIBuUMCDJG82cBAAABAQoBKBBMKLtTTeLPLbEwlOz/iewPQGlNQ6UYTvIUkJeje7If85qs3J0CAAAoYRsKHuDlx9vJFkMJyCpvPw4NqSqpeqjIoERgM8VnR7rpqU+DPQIAAAEog3S+BkMTzo1nAdRboL3bdyzOzQtRNyq7HO9f5JZ8YoaQCXdWBgEAAChLZHGLjtCT3kwtN44rPHWAlZTYWxIS+D5Q19udwr4EPwAAAAAAAAAA", 
  added_time: "1557767976382"
  }
){edges {sender, receiver, nonce, amount, fee, memo}}
}
```

## Output

```
{
  "data": {
    "AddPaymentReceipt": {
      "payment": {
        "sender": "ASgQTCi7U03izy2xMJTs/4nsD0BpTUOlGE7yFJCXo3uyH/OarNydAgAAAQ==",
        "receiver": "ASg87oNimEIn4qYlRh6XU7VgFzeX4fqjnVywWRcHiAblDAgyRvNnAQAAAQ==",
        "nonce": 2,
        "amount": "10",
        "fee": "1",
        "memo": "i!z0y����\u0011!�B5J|\u001fU�H,��\u001e\u001b%\r�\u001e���"
      }
    }
  }
}


```

# Query

## Command
```
query {user_command (filter: {
  toOrFrom: "ASgQTCi7U03izy2xMJTs/4nsD0BpTUOlGE7yFJCXo3uyH/OarNydAgAAAQ=="
}
){edges {sender, receiver, nonce, amount, fee, memo}}
}
```

## Output

```
{
  "data": {
    "payments": {
      "edges": [
        {
          "sender": "ASgQTCi7U03izy2xMJTs/4nsD0BpTUOlGE7yFJCXo3uyH/OarNydAgAAAQ==",
          "receiver": "ASg87oNimEIn4qYlRh6XU7VgFzeX4fqjnVywWRcHiAblDAgyRvNnAQAAAQ==",
          "nonce": 2,
          "amount": "10",
          "fee": "1",
          "memo": "i!z0y����\u0011!�B5J|\u001fU�H,��\u001e\u001b%\r�\u001e���"
        }
      ]
    }
  }
}
```

Checklist:

- [ ] Tests were added for the new behavior
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

### CODE REVIEW

File: src/app/cli/src/graphql.ml [hunk 1]
```diff
@@ -30,8 +30,7 @@ module Make (Commands : Coda_commands.Intf) = struct
       let public_key = Public_key.Compressed.to_base64
 
       (** Unix form of time, which is the number of milliseconds that elapsed from January 1, 1970 *)
-      let date time =
-        Time.to_span_since_epoch time |> Time.Span.to_ms |> Int64.to_string
+      let date = Time.to_string
```
Comment (bkase) [inline src/app/cli/src/graphql.ml]: can you move this comment onto `Time.to_string` in the mli

**Comment by Schmavery** [comment]
@bkase If the memo is going to be arbitrary binary data encoded in base64 then do we need a different strategy for displaying it to the user? we should talk about this if it's not just a printable string

**Comment by bkase** [comment]
@schmavery ugh you’re right

**Comment by bkase** [comment]
Given that for now we'll just handle the binary vs text data -- can you undo the base64ing, John?

**Comment by bkase** [comment]
Then we can land it

**Pull request merged**